protocol/x/clob/keeper/orders.go

```diff
@@ -961,7 +960,6 @@ func (k Keeper) AddOrderToOrderbookCollatCheck(
 	pendingUpdates := types.NewPendingUpdates()
 
 	// Retrieve the associated `PerpetualId` for the `ClobPair`.
-	oraclePriceSubticksRat := k.GetOraclePriceSubticksRat(ctx, clobPair)
 	perpetualId := clobPair.MustGetPerpetualId()
 
 	// TODO(DEC-1713): Complete as many calculations from getPessimisticCollateralCheckPrice as possible here
```

Comment:
this ticket isn't relevant anymore right? since we don't use pessimistic oracle price

Reply:
# - [ 🤌